styles.css

```diff
@@ -0,0 +1,138 @@
+/* Import */
+@import url('https://fonts.googleapis.com/css2?family=Varta&display=swap');
+
+/* Root */
+:root {
+    --primary-color: #2a9d8f;
+    --primary-dark: #023047;
+    --primary-grey: #eaeaea;
+    --primary-light-color: #e63946;
+    --white: #ffffff;
+}
+
+/* Body */
+body {
+    font-family: Varta;
+}
+
+* {
+    padding: 0;
+    margin: 0;
+}
+/* Navbar */
+.navbar {
+    padding: 1rem;
+    color: var(--primary-dark);
+    display: flex;
+    justify-content: space-around;
+}
+/* logo */
+.logo{
+    width: 5rem;
+}
+.nav-pill {
+    padding: 0.5rem;
+}
+.selected-pill {
+    font-weight: bold;
+}
+.nav-pill-link {
+    color: inherit;
+    text-decoration: none;
+}
+/* List */
+.list-style-none {
+    list-style-type: none;
+}
+.list-inline {
+    display: flex;
+}
+/* Link */ 
+.link {
+    color: var(--primary-light-color);
+    text-decoration: none;
+}
+/* Main */
+.main {
+    text-align: center;
+}
+.main p {
+    max-width: 80%;
+    margin: auto;
+    font-size: 48px;
+}
+/* Home Build */
+.hidden {
+    display: none;
+}
+.hero {
+    color: var(--primary-dark);
+    width: 100%;
+}
+.hero-heading {
+    max-width: 75%;
+    margin: auto;
+    font-size: 72px;
```

Comment:
Try not to use pixel because you end up fixing the size. Use rem for font size.
This video will help https://youtu.be/N5wpD9Ov_To

Reply:
Ya sure, I'll fix that.
And also fix the contrast problem.